docs/coding-standards.md

```diff
@@ -4,48 +4,40 @@ title: Coding Standards
 permalink: /coding-standards/
 ---
 
-Write accurate [PHPDoc](http://en.wikipedia.org/wiki/PHPDoc) styled code comments.
+## Validating with PHP Code Sniffer
 
-Write real Classes or Namespaced functions.
+Instead of reading any of this why not just let the computer nag you? From the Pressbooks plugin directory:
 
-[PHP Namespaces](https://secure.php.net/manual/en/language.namespaces.php) have been available since 2009. Namespaces are not a new concept. We use them.
+1. `composer install --dev`
```

Comment:
The `--dev` flag is no longer needed. See: https://getcomposer.org/doc/03-cli.md#install

> `--dev`: Install packages listed in `require-dev` (this is the default behavior).